plugins/lookup/conjur_variable.py

```diff
@@ -364,3 +443,14 @@ def run(self, terms, variables=None, **kwargs):
             return _store_secret_in_file(conjur_variable)
 
         return conjur_variable
+    
+    def get_var_value(self, key):
+        try:
+            variable_value = self.get_option(key)
+        except KeyError:
+            raise AnsibleError("{0} was not defined in configuration".format(key))
+
+        if not variable_value:
+            raise AnsibleError("Invalid {0} variable value".format(key))
+        
+        return variable_value
```

Comment:
Could we add some unit tests for this func? I wonder if `if not variable_value` covers all invalid values

Reply:
If the variable_value value is empy or unset then the ansible throws error.

Reply:
Suppose if the variable_value is invalid then we will get 404 http code.